README.md

```diff
@@ -12,13 +12,15 @@ This terraform module is used to integrate AWS into a meshStack instance as a me
 
 - [Terraform installed](https://learn.hashicorp.com/tutorials/terraform/install-cli)
 - [AWS CLI installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+- Three accounts to segregate accounts by function.
+  - management account: organization management account, the account that hosts the AWS Organization
+  - meshcloud account: meshStack will use this account to host the IAM users used by meshStack
+  - automation account: meshStack will use this account to manage CloudFormation that are used in [Landing Zones](https://docs.meshcloud.io/docs/meshcloud.landing-zones.html).
+- AdministratorAccess in those accounts.
 
-This module assumes you are following landing zone best practices and segregate accounts by function.
-To call this module, you will need three `aws` providers set up against different accounts
+While not strictily necessary, we recommend to configure a [terraform backend](https://developer.hashicorp.com/terraform/language/settings/backends/configuration).
```

Comment:
While not strictly necessary, we strongly recommend to configure a [terraform backend](https://developer.hashicorp.com/terraform/language/settings/backends/configuration) to store your state. Storing terraform state will allow you to maintain your meshStack integration after the initial provisioning.